src/drivers/uavcan_v1/CanardSocketCAN.cpp

```diff
@@ -39,6 +39,19 @@
 
 #include <px4_platform_common/log.h>
 
+uint64_t getMonotonicTimestampUSec(void)
+{
+	struct timespec ts;
```

Comment:
```suggestion
	struct timespec ts{};
```

instead of the memset